apps/fluent-tester/src/TestComponents/LinkExperimental/LinkTest.tsx

```diff
@@ -25,52 +26,24 @@ const DefaultLinks: React.FunctionComponent = () => {
   );
 };
 
-const InlineLinks: React.FunctionComponent = () => {
-  const doPress = React.useCallback(() => Alert.alert('Alert.', 'You have been alerted.'), []);
-  const doAllyTap = React.useCallback(() => Alert.alert('Alert.', 'You have invoked onAllyTap.'), []);
-
-  return (
-    <Stack style={stackStyle}>
-      <Text>
-        Click{' '}
-        <Link inline onPress={doPress} onAccessibilityTap={doAllyTap}>
-          this link
-        </Link>{' '}
-        to alert me.
-      </Text>
-      <Text>
-        This{' '}
-        <Link inline onPress={doPress} disabled focusable>
-          link
-        </Link>{' '}
-        is disabled but focusable.
-      </Text>
-      <Text>
-        Follow this{' '}
-        <Link inline url="https://www.bing.com/">
-          link
-        </Link>{' '}
-        to navigate.
-      </Text>
-    </Stack>
-  );
-};
-
 const SubtleLinks: React.FunctionComponent = () => {
   const doPress = React.useCallback(() => Alert.alert('Alert.', 'You have been alerted.'), []);
   const doAllyTap = React.useCallback(() => Alert.alert('Alert.', 'You have invoked onAllyTap.'), []);
+  const supportsInlineLink = Platform.OS !== ('win32' as any);
```

Comment:
I'm not sure any platform can properly support inline links for similar reasons. Maybe just... Remove it altogether?

On the macOS side, some of our collaborators at Meta have actually investigated adding an `href` prop directly to text in React Native macOS too very support links.